infinispan-remote/infinispan-server-testconfig/wildfly-trimmed-config.xml

```diff
@@ -99,6 +99,7 @@
         <subsystem xmlns="urn:infinispan:server:core:9.2">
             <cache-container name="clustered" default-cache="default" statistics="true">
                 <transport lock-timeout="60000"/>
+                <global-state/>
```

Comment:
Remember that it's nice to have in the commit also a "why" you are doing something ( not necessarly in the first line of the commit)
In this case, why do you need to add it back?

Reply:
File *wildfly-trimmed-config.xml* is a subset of cluster.xml configuration. The global state is present in all Infinispan server provided configurations (standalone.xml, clustered.xml, domain profiles) and it is necessary to allow to the server to store persistent counters data.